src/logics/Logic.cc

```diff
@@ -1265,6 +1265,24 @@ PTRef Logic::learnEqTransitivity(PTRef formula)
         return getTerm_true();
 }
 
+PTRef Logic::removeAuxVars(PTRef tr) {
+    class AuxSymbolMatcherConfig : DefaultRewriterConfig {
+        Logic const & logic;
+        bool match(PTRef tr) const {
+            if (not logic.isVar(tr)) return false; // Only variables can match
+            auto symName = std::string_view(logic.getSymName(tr));
+            return symName.compare(0, IteHandler::itePrefix.size(), IteHandler::itePrefix) == 0;
+        }
+    public:
+        AuxSymbolMatcherConfig(Logic const & logic) : logic(logic) {}
+        bool previsit(PTRef) override { return true; }
```

Comment:
Unnecessary, this is the default behaviour.

Reply:
Fixed.